runner.sh

```diff
@@ -227,6 +235,58 @@ if [ $doChecks = true ]; then
 	fi
 fi
 
+function verify_notebook_has_key_in_cell() {
+    fname=$1
+    key=$2
+    cell=$3
+    celltype=$4
+    ${PY_EXE} - << END
+import nbformat
+import re
+
+with open("$fname", "r") as f:
+    contents = nbformat.reads(f.read(), as_version=4)
+    cell = contents.cells[int("$cell")]
+    result = "true" if cell.cell_type == "$celltype" and re.search("$key", cell.source) else "false"
+print(result)
```

Comment:
this `print` doesn't seem to print anything?

Reply:
It prints string `"true"` or `"false"`, and the message in the stdout will be consumed by the following line:

```sh
if [[ $(verify_notebook_has_key_in_cell "$fname" "$license" 0 "markdown" ) != true ]];
```

If there are better ways to handle it, please let me know. Thanks @wyli !

Reply:
I see, thanks! I think it'll be more readable to have a separate python script and a bash variable to store the returned value, like this example https://github.com/Project-MONAI/model-zoo/blob/3bffe793f13a80aa648b559d3f529bb701792ac6/ci/run_regular_tests_cpu.sh#L56-L57 but the current implementation is also ok

Reply:
Thanks @wyli . The suggested way is implemented in #1152 .